libraries/botbuilder-dialogs-adaptive/src/input/choiceInput.ts

```diff
@@ -136,4 +132,17 @@ export class ChoiceInput extends InputDialog {
         return `ChoiceInput[${ this.prompt && this.prompt.toString() }]`;
     }
 
+    private determineCulture(dc: DialogContext, opt: FindChoicesOptions = null): string {
```

Comment:
Why are we defaulting to `null`? It seems unnecessary as the default, even if the parameter is omitted, is `undefined` which is falsey. This happens in a few places that can all be cleaned up.

Reply:
TypeScript is unhappy with omitted parameters unless they have a default value. Removing the default value will result in a TypeScript compiler error. Is it better to pass null as an argument to the function call instead of as a default value?

Reply:
Discussed on Teams that using `?` to mark the parameter as optional will work.

Reply:
Never mind, I forgot about `?` syntax